docs/Api/OAuthApi.md

```diff
@@ -0,0 +1,117 @@
+# Kinde\KindeSDK\OAuthApi
+
+All URIs are relative to https://app.kinde.com/api/v1.
```

Comment:
@trunges21 The `user_profile` endpoints are relative to just `https://{subdomain{.kinde.com/`. This probably has to be changed in the generator.